rust/src/impl_mockchain/key.rs

```diff
@@ -0,0 +1,321 @@
+//! Module provides cryptographic utilities and types related to
```

Comment:
Do we actually use anything in this file? Everything in here is Jormungandr-specific so we shouldn't need any of it.